presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java

```diff
@@ -61,6 +61,7 @@
     public static final String TLS = "tls";
     public static final String CANCEL_QUERY = "cancel_query";
     public static final String BIG_QUERY = "big_query";
+    public static final String HIVE_TABLE_STATISTICS = "hive_table_statistics";
```

Comment:
i don't think we need this test group (but i don't have strong opinion)

Reply:
Unfortunately it is problematic to run the tests by a class name (at least as i know). I added it to be able to run only this subset of tests. @kokosing can we run a subset of tests by specifying a class name?